source/modules/collection-api/account/actions/create.md.erb

```diff
@@ -1,28 +1,24 @@
 ### Account Creation
-
-```php
-<?php 
-
-use Inspetor; 
-
-$inspetor = $this->getConfiguredInspetor(); 
-
-$inspetor->trackAccountCreation($account); 
-
-?>
+```sh
+curl -XPOST https://collection-prod.inspcdn.net/account \
+    -H 'Authorization: Bearer <your API key>'
+    -d '{
+        "id":            "12345",
+        "email":         "my_user@inspetor.com",
+        "document":      "123.456.789-10",
+        "phone_number":  "11999123456",
```

Comment:
We are missing the country code here

Reply:
feels like country code should actually be an additional field, no?

Reply:
Why? It's part of your cellphone number